src/ray/raylet/scheduling/fixed_point.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020-2021 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ray/raylet/scheduling/fixed_point.h"
+
+#include <sstream>
+
+bool FixedPointVectorEqual(const std::vector<FixedPoint> &v1,
```

Comment:
Won't `v1 == v2` just work?

Reply:
Yeah, I just moved it from another file. Will remove it.